internal/slave/service.go

```diff
@@ -142,16 +173,26 @@ func (ss *slaveService) pollAndApplyChanges() {
 
 func (ss *slaveService) applyChangesFromMaster(chngsPerBatch uint32) error {
 	ss.lg.Info("Retrieving changes from master", zap.Uint64("FromChangeNumber", ss.fromChngNum), zap.Uint32("ChangesPerBatch", chngsPerBatch))
+
+	if ss.replCli == nil {
+		if err := ss.findAndConnectToMaster(); err != nil {
+			return errors.New("Can not replicate as replication client not yet established")
```

Comment:
Pls do a backoff retry in-this case. Otherwise leads to bombardment of retry
![image](https://user-images.githubusercontent.com/709368/127199635-40991aec-8263-4f0f-8c88-47dffdccee42.png)

Reply:
this would be attempted on every replication attempt - wrt poll interval. the reason its bomboarding is because poll interval was set to 100 ms in ProcFile. set it to 2 secs. that fixes it. there is anyways a sleep of 5 secs in our CI pipeline after initiating ProcFile so won't cause any delay.

Reply:
Sleep inside a procfile is not possible. Instead You can split it into multiple procfile and load in sequence.

Reply:
yea not sleeping inside procfile. i meant changing the poll interval from 100ms to 5 seconds in ProcFile .